ios/brave-ios/Sources/Brave/Frontend/Rewards/BraveRewards.swift

```diff
@@ -152,12 +152,11 @@ public class BraveRewards: PreferencesObserver {
   }
 
   public var shouldStartAds: Bool {
```

Comment:
Either keep this and also keep the usage of it in BrowserViewController.swift or delete this and its usages